content/sensu-go/5.14/guides/clustering.md

```diff
@@ -183,6 +183,50 @@ sensuctl cluster member-remove 2f7ae42c315f8c2d
 Removed member 2f7ae42c315f8c2d from cluster
 {{< /highlight >}}
 
+### Replace a faulty cluster member
+
+Here's how to replace a faulty cluster member to restore a cluster's health.
+
+First, run `sensuctl cluster health` to identify the faulty cluster member.
+For a faulty cluster member, the `Error` column will include an error message and the `Healthy` column will list `false`.
+
+In this example, cluster member `backend-4` is faulty:
+
+{{< highlight shell >}}
+sensuctl cluster health
+
+       ID            Name                          Error                           Healthy  
+────────────────── ─────────── ─────────────────────────────────────────────────── ─────────
+a32e8f613b529ad4   backend-1                                                        true
+c3d9f4b8d0dd1ac9   backend-2                                                        true
+c8f63ae435a5e6bf   backend-3                                                        true
+2f7ae42c315f8c2d   backend-4  dial tcp 10.0.0.4:2379: connect: connection refused   false
+
+{{< /highlight >}}
+
+Second, delete the faulty cluster member. To continue this example, you will delete cluster member `backend-4`:
+
+{{< highlight shell >}}
+sensuctl cluster member-remove 2f7ae42c315f8c2d
+
+Removed member 2f7ae42c315f8c2d from cluster
+{{< /highlight >}}
+
+Third, add the faulty cluster member back to the cluster. Make sure to change the `etcd-initial-cluster-state` to `existing` in the configuration.
```

Comment:
I am not quite sure of the language to use here, but we don't want to say "add the faulty cluster member back". While it is true you can use the same name, same IP for the new cluster member, in practice that server will be completely different; the old one is gone as there is something wrong with it.

So we need something along the lines of "Third, add a newly made server to the cluster. It can be named the same as the faulty one, with the same IP address and mostly the same configuration. Before adding, make sure to change the `etcd-initial-cluster-state` to `existing` in the `backend.yaml` configuration file."